src/qforte/sq_op_pool.h

```diff
@@ -27,9 +30,6 @@ class SQOpPool {
     ///     of occupied spin orbitals of each point group symmetry
     void set_orb_spaces(const std::vector<int>& ref);
 
-    /// returns the JW transformed vector of operators
-    std::vector<QuantumOperator> get_quantum_operators();
-
     /// returns a QuantumOpPool object with one term for each term in terms_
     QuantumOpPool get_quantum_op_pool();
```

Comment:
I think that we will have to do some renaming in the future. First, remove all the redundant `Quantum`. However, `QuantumOperator` should become `QubitOperator`.

Reply:
And so this function should become `get_qubit_op_pool`.

Reply:
Or better, `qubit_op_pool`.